admin-ui/plugins/auth-server/redux/sagas/OAuthScopeSaga.js

```diff
@@ -6,220 +6,217 @@ import {
   takeEvery,
   takeLatest,
   select,
-} from 'redux-saga/effects'
-import { getAPIAccessToken } from '../actions/AuthActions'
-import { updateToast } from 'Redux/actions/ToastAction'
-import {
-  getScopesResponse,
-  getScopeByPatternResponse,
-  addScopeResponse,
-  editScopeResponse,
-  deleteScopeResponse,
-  getScopeByCreatorResponse,
-  setCurrentItem,
-  getClientScopesResponse,
-} from '../actions/ScopeActions'
-import {
-  GET_SCOPES,
-  SEARCH_SCOPES,
-  GET_SCOPE_BY_INUM,
-  ADD_SCOPE,
-  EDIT_SCOPE,
-  DELETE_SCOPE,
-  GET_SCOPE_BY_PATTERN,
-  GET_SCOPE_BY_CREATOR,
-  GET_CLIENT_SCOPES,
-} from '../actions/types'
-import { SCOPE } from '../audit/Resources'
+} from "redux-saga/effects";
+import { getAPIAccessToken } from "../actions/AuthActions";
+import { updateToast } from "Redux/actions/ToastAction";
+import { SCOPE } from "../audit/Resources";
 import {
   CREATE,
   UPDATE,
   DELETION,
   FETCH,
-} from '../../../../app/audit/UserActionType'
-import ScopeApi from '../api/ScopeApi'
-import { getClient } from 'Redux/api/base'
-import { isFourZeroOneError, addAdditionalData } from 'Utils/TokenController'
-import { postUserAction } from 'Redux/api/backend-api'
+} from "../../../../app/audit/UserActionType";
```

Comment:
semi colon is not used everywhere after the sentence ending. We should maintain uniformity.